forks/artifacts/Dockerfile.centos9

```diff
@@ -0,0 +1,24 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS oc
+ARG OS_GIT_VERSION="4.7.0"
```

Comment:
why is this not 4.12 here?

Reply:
I set the value to be the same as our default one in Prow. Changed now.